### ISSUE TITLE
Add GraphQL docstrings

### ISSUE THREAD

**Issue by Waidhoferj** [Waidhoferj/CSC-308-309-Project#61]
Add docstrings to our `api_types.py` objects. The docstring will show up in the documentation side bar of the graphQL interface and make it easy for everyone to understand what each object does in plain english. 

For each object, do something like this:
```
class UserType(MongoengineObjectType):
"""
GeoArt user profile...
"""
...
```

And ensure this shows up in the GraphQL documentation sidebar.

**Comment by Braden50** [comment]
Actual time: 1 hour
